snet_cli/commands.py

```diff
@@ -277,10 +283,13 @@ def transact(self):
 
 
 class OrganizationCommand(BlockchainCommand):
-    def _getorganizationbyname(self):
-        return self.call_contract_command("Registry", "getOrganizationByName", [type_converter("bytes32")(self.args.name)])
+    def _getorganizationbyid(self, org_id):
+        org_id_bytes32 = type_converter("bytes32")(org_id)
+        if (len(org_id_bytes32) > 32):
+            raise Exception("Your org_id is too long: len(org_id_bytes32)=%i"%(len(org_id_bytes32)))
```

Comment:
Lets include the allowed length of 32 bytes in the message something like below.
`raise Exception("Your org_id is too long, it should be 32 bytes or less. len(org_id_bytes32)=%i"%(len(org_id_bytes32)))`